### ISSUE TITLE
does not work with scss-lint 0.26.0

### ISSUE THREAD

**Issue by biophonc** [ahmednuaman/grunt-scss-lint#57]
Hi,

please add a note to the README that it does not work with the above mentioned version, or update your plugin if you like to - depends on what you prefer :)

See the change list: https://github.com/causes/scss-lint/releases

scss-lint has now new error codes: 

```
Change CLI to return exit code of 1 if only warnings are reported, 
or 2 if any errors are reported (was previously 65 for either)
```

Means you don't want to check for code `65` but for `2`.

Also the option `TrailingSemicolonAfterPropertyValue` was renamed to `TrailingSemicolon`. 

Thanks!
